frontend/components/forms/ConfirmInviteForm/ConfirmInviteForm.jsx

```diff
@@ -28,38 +28,37 @@ class ConfirmInviteForm extends Component {
     return (
```

Comment:
@lukeheath What's the correct procedure to make a tech debt ticket to convert these last few jsx forms to our new tsx forms?

Reply:
Found an issue related to converting jsx to tsx: https://github.com/fleetdm/fleet/issues/8980

Reply:
to be clear, this is unrelated to this PR right

Reply:
@jacobshandling totes